src/Whoops/Handler/Handler.php

```diff
@@ -14,13 +14,20 @@
  */
 abstract class Handler implements HandlerInterface
 {
-    /**
-     * Return constants that can be returned from Handler::handle
-     * to message the handler walker.
+    /*
+     Return constants that can be returned from Handler::handle
+     to message the handler walker.
      */
     const DONE         = 0x10; // returning this is optional, only exists for
                                // semantic purposes
+    /**
+     * The Handler has handled the Throwable in some way, and wishes to skip any other Handler.
+     * Execution will continue.
+     */
     const LAST_HANDLER = 0x20;
```

Comment:
this constant somehow confused me.

I am in the process of extending the PrettyPageHandler and actually want to ignore some constants which get thrown.
"Ignore" in this context means I dont want to handle it and I dont want whoops to generate output.

I could not find a way yet how this can be achieved

Reply:
ohh my problem is that Exception cannot be thrown, catched by the exception handler and control-flow goes on... this only works with errors/warnings/infos.